draft-ietf-jsonpath-base.md

```diff
@@ -659,7 +666,7 @@ The syntax and semantics of each kind of selector are defined below.
 A name selector `'<name>'` selects at most one object member value.
 
 In contrast to JSON,
-the JSONPath syntax allows strings to be enclosed in _single_ or _double_ quotes.
+the JSONPath syntax allows strings to be enclosed in *single* or *double* quotes.
```

Comment:
This is not a change (two markdown syntaxes for the same thing), or is it?

Reply:
No, you're right. Reinstated.